demucs/separate.py

```diff
@@ -127,13 +133,29 @@ def main():
 
         track_folder = out / track.name.rsplit(".", 1)[0]
         track_folder.mkdir(exist_ok=True)
-        for source, name in zip(sources, model.sources):
-            stem = str(track_folder / name)
+        if args.stem == None:
+            for source, name in zip(sources, model.sources):
+                stem = str(track_folder / name)
+                if args.mp3:
+                    stem += ".mp3"
+                else:
+                    stem += ".wav"
+                save_audio(source, stem, model.samplerate)
+        else:
+            stem = str(track_folder / args.stem)
+            if args.mp3:
+                stem += ".mp3"
+            else:
+                stem += ".wav"
+            save_audio(sources.pop(model.souces.index(args.stem)), stem, model.samplerate)
+            # Warning : after poping the stem, selected stem is no longer in the list 'sources'
+            other_stem = sum(i.numpy() for i in sources)
```

Comment:
I don't think you need the `numpy` here no ?

Reply:
I don't sure whether torch can add them directly, so I converted it into jumpy first.

Reply:
it will work fine without the convertion to numpy :)